openapi/index.yaml

```diff
@@ -93,6 +94,7 @@ paths:
         Get the EYCA status details by the provided fiscal code. 
         In case of success the response could be one of:
           - CardPending
```

Comment:
Isn't the case to refactor to EycaCardPending?

Reply:
Nope, CardPending is a status inherited from CommonCard